lua/complementree/utils.lua

```diff
@@ -13,4 +14,58 @@ function M.cword(complete_item)
   return (complete_item.word or complete_item.abbr)
 end
 
+M.make_relative_path = function(path, root)
+  local baselen = #root
+  if path:sub(0, baselen) == root then
+      path = path:sub(baselen+2)
+  end
+  return path
+end
+
+local function get_os_file_separator()
+  local os = string.lower(jit.os)
+  if os == "linux" or os == "osx" or os == "bsd" then
+    return "/"
+  else
+    return "\\"
+  end
+end
+
+local os_sep = get_os_file_separator()
```

Comment:
As you do that only once, maybe avoid the function call and set `os_sep` correctly directly ?